packages/website/src/docs/form/select-menu.mdx

```diff
@@ -649,6 +649,29 @@ function Example() {
 }
 ```
 
+### Sizes
+
+```function-live
+function Example() {
+  const [value, setValue] = React.useState();
+
+  return (
+    <SelectMenu
```

Comment:
Nice! Maybe we can add the `medium` and `large` variant to demonstrate different sizes?

Reply:
@jxom done. I split the size variants into their own sections, let me know if that doesn't suit